README.md

```diff
@@ -26,10 +26,16 @@ To build an individual package, run the following command:
 yarn build
 ```
 
-To build an individual package in response to changes within the `src` directory, run the following command:
+The following command will generate documentation for each module in the `packages` directory. It uses [TypeDoc](https://typedoc.org/) and outputs to the `docs` directory.
+
 ```sh
 yarn dev
 ```
+##Documentation
+The following command will generate documentation in the `docs` directory for each module in the `packages` directory using [TypeDoc](https://typedoc.org/).
```

Comment:
Use @DanielMoniz-GBI's suggestion here instead.

Reply:
that is his suggestion... https://groupbyinc.slack.com/archives/CHUMB6UM6/p1561401593150300?thread_ts=1561399180.149300&cid=CHUMB6UM6

Reply:
@jmbtutor the suggestion applied to the wrong line. I applied the suggestion to the correct line, and fixed the wrong line it had overridden. If you look at the latest version of readme it should all be fixed :)